### PR TITLE
feat: copy regions chart

### PR DESCRIPTION
**Pull request by Dav-14**
FIXES PLA-46

### CODE REVIEW

File: charts/regions/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+dependencies:
+  - name: agent
+    version: v2.1.0
+    repository: "file://../agent"
+    condition: agent.enabled
+  - name: operator
+    version: v2.0.19
+    repository: "oci://ghcr.io/formancehq/helm"
+    condition: operator.enabled
```
Comment (Dav-14) [inline charts/regions/Chart.yaml:25]: On the last release

File: charts/regions/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+type: application
+version: "v2.1.0"
+appVersion: "latest"
+
+dependencies:
+  - name: agent
+    version: v2.1.0
+    repository: "file://../agent"
+    condition: agent.enabled
```
Comment (Dav-14) [inline charts/regions/Chart.yaml:21]: On the local release

File: charts/regions/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+sources:
+  - "https://github.com/formancehq/operator"
+  - "https://github.com/formancehq/agent"
+maintainers:
+  - name: "Formance Team"
+    email: "support@formance.com"
+icon: "https://avatars.githubusercontent.com/u/84325077?s=200&v=4"
+
+type: application
+version: "v2.1.0"
```
Comment (Dav-14) [inline charts/regions/Chart.yaml:14]: Not in beta anymore

File: charts/regions/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+appVersion: "latest"
+
+dependencies:
+  - name: agent
+    version: v2.1.0
+    repository: "file://../agent"
```
Comment (flemzord) [inline charts/regions/Chart.yaml:20]: Use OCI release ?

Comment (Dav-14) [reply charts/regions/Chart.yaml:20]: Not needed as agent

**Pull request merged**